src/pck/mod.rs

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: Apache-2.0
+
+mod raw;
+
+use raw::SgxExtensionRaw;
+
+use const_oid::ObjectIdentifier;
+use der::Decode;
+use x509::ext::Extensions;
+
+const SGX_EXTENSION: ObjectIdentifier = ObjectIdentifier::new_unwrap("1.2.840.113741.1.13.1");
```

Comment:
use `SgxExtensionRaw::OID` instead of this?

Reply:
Done